python-sdk/conftest.py

```diff
@@ -81,13 +81,7 @@ def database_table_fixture(request):
     database.populate_table_metadata(table)
     database.create_schema_if_needed(table.metadata.schema)
 
-    kwargs = {}
```

Comment:
Why did we change this?

Reply:
Yeah I'd also be interested in why we changed this

Reply:
@dimberman @tatiana I have added the Load_option code as part of the Database and File/Location and File/Type classed because of which can pass the load_option as initialization of those classes. Hence we can remove it from this place and others. There are still places we can remove such code, I was thinking to do this in a separate PR.